src/styles/components/details.scss

```diff
@@ -0,0 +1,31 @@
+.price__price {
+  color: $secondary-color;
+  font-family: Roboto;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 18px;
+}
+
+.price__dollar {
+  color: $text-color;
+  text-align: right;
+  font-family: Roboto;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 18px;
+}
+
+.review__5 {
+  justify-content: center;
+  align-self: center;
+  color: $text-color;
+  text-align: right;
+  font-family: Roboto;
+  font-size: 10px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 14px;
+  padding-top: 1px;
+}
```

Comment:
The file has name "details.scss" but no details block was found 🤷🏻‍♂️

For this task you need only "stars" and "card" BEM blocks. Checklist:
6. [BEM] - Don't make BEM structure too complicated. You don't need to create separate blocks for anything other than card itself and stars.